web-registry/src/lib/clients/graphqlClient.ts

```diff
@@ -0,0 +1,8 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client';
```

Comment:
we could move `sanity.ts` to this subfolder as well
also sometimes we use `apolloClient` and sometimes `graphqlClient`, maybe instead we could just use `pgClient` (similarly to `sanityClient`) to indicate it's the postgres db client without necessarily saying it's a graphql/apollo client

Reply:
on the other hand, we have a folder named `lib/queries/react-query/registry-server/graphql` with queries using this client so maybe it's just best to keep `graphqlClient` everywhere...

Reply:
Fair points, will move sanity to the client folder.
Regarding `apolloClient` and `graphqlClient `:
- `graphqlClient`: is an in-memory apollo client (mostly used in loader)
- `apolloClient`: is the apollo client coming from the context.

The first one is more like a generic graphql client (hence the name) while the second one comes from `apollo.tsx` file.
Maybe we should use only `graphqlClient` as a convention if we don't care about where the client comes from.

Reply:
yeah, `apolloClient` coming from the context is a bit different since under the hood, it adds some Authorization headers to requests but that was based on the "legacy" login system which is not really used anymore and I'm not sure if we'll have a similar strategy with the new Keplr based login system
so I agree that for now, it's fine to just use `graphqlClient` everywhere we use react-query since the legacy login system is not used

Reply:
Looks fine to me. I just changed the code related to this PR to use only `graphqlClient`.

Reply:
ok I will make a small PR to do the same on Storefront related queries